===================================================================
RCS file: /cvsroot/gphoto/libgphoto2/camlibs/canon/canon.c,v
retrieving revision 1.240
retrieving revision 1.241
diff -u -r1.240 -r1.241
--- gphoto/libgphoto2/camlibs/canon/canon.c	2004/03/07 20:41:58	1.240
+++ gphoto/libgphoto2/camlibs/canon/canon.c	2004/04/04 10:21:51	1.241
@@ -88,12 +88,12 @@
  **/
 
 /* SL_* - size limit constants */
-#define KILOBYTE	(1024)
-#define MEGABYTE	(1024 * KILOBYTE)
-#define SL_THUMB	( 100 * KILOBYTE)
-#define SL_PICTURE	(  10 * MEGABYTE)
-#define SL_MOVIE_SMALL	( 100 * MEGABYTE)
-#define SL_MOVIE_LARGE	(2048 * MEGABYTE)
+#define KILOBYTE	(1024U)
+#define MEGABYTE	(1024U * KILOBYTE)
+#define SL_THUMB	( 100U * KILOBYTE)
+#define SL_PICTURE	(  10U * MEGABYTE)
+#define SL_MOVIE_SMALL	( 100U * MEGABYTE)
+#define SL_MOVIE_LARGE	(2048U * MEGABYTE)
 #define NO_USB  0
 
 const struct canonCamModelData models[] = {
===================================================================
RCS file: /cvsroot/gphoto/libgphoto2/camlibs/canon/usb.c,v
retrieving revision 1.83
retrieving revision 1.85
diff -u -r1.83 -r1.85
--- gphoto/libgphoto2/camlibs/canon/usb.c	2004/01/02 15:42:33	1.83
+++ gphoto/libgphoto2/camlibs/canon/usb.c	2004/02/03 18:24:39	1.85
@@ -1178,7 +1178,7 @@
 
 	/* the 1 is to show status */
 	res = canon_usb_long_dialogue (camera, CANON_USB_FUNCTION_GET_FILE, data, length,
-				       camera->pl->md->max_picture_size, payload,
+				       camera->pl->md->max_movie_size, payload,
 				       payload_length, 1, context);
 	if (res != GP_OK) {
 		GP_DEBUG ("canon_usb_get_file: canon_usb_long_dialogue() "

